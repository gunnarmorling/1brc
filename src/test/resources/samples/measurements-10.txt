Halifax;12.9
Zagreb;12.2
Cabo San Lucas;14.9
Adelaide;15.0
SÃ©gou;25.7
Pittsburgh;9.7
Karachi;15.4
Xi'an;24.2
Dodoma;22.2
Tauranga;38.2
