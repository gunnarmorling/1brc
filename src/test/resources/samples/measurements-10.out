{Adelaide=15.0/15.0/15.0, Cabo San Lucas=14.9/14.9/14.9, Dodoma=22.2/22.2/22.2, Halifax=12.9/12.9/12.9, Karachi=15.4/15.4/15.4, Pittsburgh=9.7/9.7/9.7, SÃ©gou=25.7/25.7/25.7, Tauranga=38.2/38.2/38.2, Xi'an=24.2/24.2/24.2, Zagreb=12.2/12.2/12.2}
